src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {useDispatch, useSelector} from 'react-redux';
+import {useQuery} from '@apollo/react-hooks';
+
+import {cmGoto} from '~/JContent.redux-actions';
+import {GetContentPath} from './ContentPath.gql-queries';
+import ContentPath from './ContentPath';
+
+const ContentPathContainer = ({path}) => {
+    const dispatch = useDispatch();
+
+    const {mode, language} = useSelector(state => ({
+        mode: state.jcontent.mode,
+        language: state.language
+    }));
+
+    const {data, error} = useQuery(GetContentPath, {
+        variables: {
+            path: path,
+            language
+        }
+    });
+
+    const handleNavigation = path => {
+        dispatch(cmGoto({
+            mode: mode || 'content-folders',
```

Comment:
What happen if `cmGoto` is not available (like content editor loaded without jContent?)
We need to handle that case

Reply:
if there is no jContent, we will not be possible to redirect content anywhere

Reply:
if `cmGoto` not present, this call will fail, you need to be sure that cmGoto is available before building the link.

Reply:
I'll create task to handle this case